test/addons-napi/test_dataview/test.js

```diff
@@ -10,5 +10,4 @@ const buffer = new ArrayBuffer(128);
 const template = Reflect.construct(DataView, [buffer]);
 
 const theDataview = test_dataview.CreateDataView(template);
-assert.ok(theDataview instanceof DataView,
-          'The new variable should be of type Dataview');
+assert.ok(`Expect ${theDataview} to be a DataView`);
```

Comment:
This is no longer correct as it is always truthy. Can you please add back the first argument? `theDataview instanceof DataView`.

Reply:
Yep, sorry. Shouldn't do things while asleep.

Reply:
No problem. Now I think the line is too long and our linter isn't happy with it. Can you please move the second argument to the next as it was originally?

Thanks!